index.html

```diff
@@ -60,40 +60,62 @@
 				<br />
 			</p>
 			<p class="moreinfo">
-				Use the drop-down menus below to zoom to a <span id="house-district-reference">House</span><span id="senate-district-reference">Senate</span> District of your choice.
+				Use the drop-down menus below to zoom to a <span id="house-district-reference">House</span><span id="senate-district-reference">Senate</span><span id="school-district-reference">School</span> District of your choice.
 				<br /><br />
 
 				<!--Drop down controls-->
 
 				<select id="house-districts-control" onchange="zoomToPolygon('state-house-districts', this.value, 'house_dist');"></select>
 				<select id="senate-districts-control" onchange="zoomToPolygon('state-senate-districts', this.value, 'senate_dist');"></select>
+				<select id="school-districts-control" onchange="zoomToPolygon('state-school-districts', this.value, 'NAME', maskLayer=false);"></select>
```

Comment:
`maskLayer=false` here is what tells `zoomToPolygon` to skip all the filtering and selective hiding for school districts.